src/app/components/ResourceHints/index.jsx

```diff
@@ -1,24 +1,32 @@
-import React from 'react';
-import { arrayOf, string } from 'prop-types';
+import React, { Fragment } from 'react';
+import { any, objectOf, string } from 'prop-types';
 
-const ResourceHints = ({ assetOrigins }) => (
-  <>
-    {assetOrigins.map(origin => (
-      <link
-        rel="preconnect"
-        key={origin}
-        href={origin}
-        crossOrigin="anonymous"
-      />
-    ))}
-    {assetOrigins.map(origin => (
-      <link rel="dns-prefetch" key={origin} href={origin} />
-    ))}
-  </>
-);
+const ResourceHints = ({ assetOrigins, service }) => {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const serviceConfig = require(`../../lib/config/services/${service}`);
+  const { fonts } = serviceConfig.service.default;
+  const { fontsOrigins, ...rest } = assetOrigins;
+  let origins = Object.values(rest).flat();
+
+  if (fonts && fonts.length > 0) {
```

Comment:
Fonts currently can come from 2 locations `ws-downloads.files.bbci.co.uk` and `gel.files.bbci.co.uk`, ideally we should be able to only preload the one we need. Feel free to add a `fontAssetOrigins` key to service configs

Reply:
@dr3 I think this will be handled as part of [this PR](https://github.com/bbc/simorgh/issues/3779). From [this](https://github.com/bbc/simorgh/issues/3779#issue-494171756) description I gather we'll soon be moving all fonts to `ws-downloads`